apache/superset
# feat: Selectable dropdown triggered by icon component

### SUMMARY
This PR implements a selectable dropdown menu that can be opened through an icon. This is useful for the implementation of the horizontal filter bar and has been included in the header of the filter bar for that reason.

### AFTER

![Screenshot 2022-10-27 at 19 07 48](https://user-images.githubusercontent.com/60598000/198347982-5ebf65cd-3cfc-4970-a8a4-96c961ce1562.png)

### TESTING INSTRUCTIONS
1. Open the filter bar in a Dashboard
2. Click on the gear icon
3. The dropdown should appear with two options, vertical and horizontal

### ADDITIONAL INFORMATION
